static/scss/dashboard.scss

```diff
@@ -251,3 +251,25 @@ c.validation-wrapper {
 .education-icons {
   text-align: right;
 }
+
+.terms-of-service-main {
+  width: 75%;
+  margin-left: auto;
+  margin-right: auto;
+  div.terms-card-header {
+    padding: 25px 0 25px 0;
+    font-size: x-large;
+    color: #b34a4c;
+    font-weight: bold;
+  }
+  .terms-of-service-container {
+    height: 500px;
+    overflow-y: scroll;
+    li {
+      display: list-item;
+    }
+    .terms-of-service-content {
+      padding: 7px;
```

Comment:
Padding here should be similar to title, 20px I think? 

here's a screenshot to help explain: 
![screenshot-localhost 8079 2016-05-20 08-24-18](https://cloud.githubusercontent.com/assets/430126/15427640/81f343a0-1e64-11e6-8b30-4b67bff178de.png)